app/src/main/res/values/attrs.xml

```diff
@@ -7,4 +7,23 @@
   <attr name="defaultBoldTypeface" format="string" />
   <attr name="defaultLightTypeface" format="string" />
 
-</resources>
\ No newline at end of file
+  <declare-styleable name="CardLayout">
+    <attr name="cardCornerRadius" format="dimension" />
+    <attr name="cardInsetHorizontal" format="dimension" />
+    <attr name="cardInsetTop" format="dimension" />
+    <attr name="cardInsetBottom" format="dimension" />
+  </declare-styleable>
+
+  <attr name="cardViewDefaultStyle" format="reference" />
+
+  <style name="Widget.Squanchy.CardLayout" parent="None">
```

Comment:
That's a default style (`Widget.`), didn't want to mix it with "real" ones. If we end up having more base styles I'll put them in a separate file